server/graphql/v2/query/LatestChangelogPublishDateQuery.js

```diff
@@ -0,0 +1,31 @@
+import { GraphQLDateTime } from 'graphql-iso-date';
+
+import cache, { fetchCollectiveId } from '../../../lib/cache';
+import sequelize from '../../../lib/sequelize';
+import models from '../../../models';
+
+const LatestChangelogPublishDateQuery = {
+  type: GraphQLDateTime,
+  async resolve() {
+    const cacheKey = 'latest_changelog_publish_date';
+    let latestChangelogPublishDate = await cache.get(cacheKey);
+    if (!latestChangelogPublishDate) {
+      const collectiveId = await fetchCollectiveId('opencollective');
```

Comment:
I forgot, are we supporting posts to `engineering` and `design` collectives at this stage?

Reply:
@znarf : The plan is on this issue to support posts from `opencollective`; the task for supporting the `engineering` and `design` collectives will be handled at a later issue; this probably needs reviewing I think, but I'll create an issue for it soon. 🐨